### DISCUSSION TITLE
Mini pairs not working correctly by default?

### DISCUSSION THREAD

**Discussion by adamsitar** [LazyVim/LazyVim#4888] [Q&A] [answered]
When I attempt the "move to previous empty line" (so with `{`), the autopair plugin triggers even if it's in command mode, therefore preventing the move.

Does anyone have this issue? Or is this being caused by something in my own config?

**Comment by dpetka2001** [comment, marked as answer]
I can't repro with default LazyVim installation. Screencast following showing default LazyVim installation from scratch

[Screencast_2024-11-25-16-42-18.webm](https://github.com/user-attachments/assets/d397b90b-9154-4f58-8667-e7998eb9ac8b)

I'm cloning from my own fork because I've added `showkeys` plugin which is helpful for showing what keys I type. The branch can be found [here](https://github.com/dpetka2001/starter/tree/feat/showkeys) and compared with the default LazyVim starter installation, so that everyone can see that the only additional thing is the installation of the plugin `showkeys`.

**Reply by adamsitar** [reply]
thanks, it works in the default lazyvim, I managed to find the issuse in my config too
